Sources/FileSystem/Model/BetaGroup.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+
+public struct BetaGroup: Codable, Equatable {
+
+    public typealias EmailAddress = String
```

Comment:
Maybe we want to think about having some sort of strongly typed email in our `Model` module